packages/openapi-fetch/README.md

```diff
@@ -16,18 +16,18 @@ The syntax is inspired by popular libraries like react-query or Apollo client, b
 import createClient from "openapi-fetch";
 import type { paths } from "./api/v1"; // generated by openapi-typescript
 
-const { GET, PUT } = createClient<paths>({ baseUrl: "https://myapi.dev/v1/" });
+const client = createClient<paths>({ baseUrl: "https://myapi.dev/v1/" });
 
 const {
   data, // only present if 2XX response
   error, // only present if 4XX or 5XX response
-} = await GET("/blogposts/{post_id}", {
+} = await client.GET("/blogposts/{post_id}", {
```

Comment:
Extremely in-the-weeds: when writing the docs for middleware, I found `const { use } = createClient()` to be an awkward syntax (it it just me or does a floating `use()` just seem weird?). Renaming it (`const { use: useMiddleware }`) also felt awkward. `client.use(…)` felt like the most natural syntax, and matched other libraries.

As a result, it felt a little weird in other places to both keep `client` around and destructure it, so I opted to simplify and just keep `client` around in all places. Really, there’s no recommended syntax, and people can do what they want. I’m just trying to write the docs in the most “copy-pasteable” way :shrug: